webpack.config.cjs

```diff
@@ -230,6 +230,87 @@ const storyEmbedBlock = {
   },
 };
 
+const webStoriesBlock = {
```

Comment:
I am not sure why so much is being overridden here. Can't be just copy something like this to be as much of the config shared. 

https://github.com/rtCamp/web-stories-wp/blob/e1138ce9d0eed8f0d41230cf7232a58943e57833/webpack.config.cjs#L314-L339

Reply:
For the `module.rules` i've answered query below. 

I've used the sharedConfig where it's possible, upon taking another look I can possibly remove just two entries from the `plugins` with `...sharedConfig.plugins`. I'll update that.

Reply:
removed unnecessary configs.